pkgs/development/tools/build-managers/cmake/setup-hook.sh

```diff
@@ -25,8 +25,8 @@ cmakeConfigurePhase() {
     fi
 
     if [ -z "${dontUseCmakeBuildDir-}" ]; then
-        mkdir -p build
-        cd build
+        mkdir -p ${cmakeBuildDir:-build}
+        cd ${cmakeBuildDir:-build}
```

Comment:
Just a suggestion.

```suggestion
        cmakeBuildDir=${cmakeBuildDir:-build}
        mkdir -p ${cmakeBuildDir}
        cd ${cmakeBuildDir}
```

I believe it conveys better the idea of "if not set, set cmakeBuildDir to this default value"; also, being a variable that is used multiple times, it is better to concentrate its setting to one place.

Reply:
Following the convention in the stdenv, it should be:

```
: ${cmakeBuildDir:=build}
```

See: https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/generic/setup.sh#L1010

Reply:
I like it